microsoftgraph/microsoft-graph-explorer-v4
# Fix: Autocomplete character selection

## Overview
#2298 

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
1. Check out this branch
2. click in the query bar
3. type: /messages?$sel
4. Note that $select is suggested
5. Press enter
6. Query bar shows https://graph.microsoft.com/v1.0/me?$select=
